hw/ip/aes/data/aes.hjson

```diff
@@ -87,7 +87,9 @@
       hwre:     "true",
       fields: [
         { bits: "31:0", name: "data_out", desc: "Output Data" }
-      ],
+      ]
```

Comment:
@rasmus-madsen 
Just a heads up - I went ahead and fixed this, so you don't have to. I wanted to get the chip level CSR tests passing.